CommunityFeatures.md

```diff
@@ -0,0 +1,40 @@
+# Introduction
+
+Every time a Pull request improves the community firmware it shall note down it's achievements and usage in this document.
+
+
+# General improvements
+
+Here is a list of general improvements that have been made as a list ordere by date:
+
+* 2023-06-17 - This list was started and it's first entry shall be replaced with the first improvement

```

Comment:
Rather than date, it might be more useful to link the PR that implements the feature. Linking the PR has the added benefit of giving some credit to the original implementer =)